otter/convergence/steps.py

```diff
@@ -558,3 +558,17 @@ def _rcv3_check_bulk_delete(attempted_pairs, result):
         return next_step.as_effect()
     else:
         return StepResult.SUCCESS, []
+
+
+@implementer(IStep)
+@attributes([Attribute('seconds', instance_of=float)])
```

Comment:
I was thinking on similar lines. Just wanted another eyes to have a look at it :) Will change it.